### ISSUE TITLE
Improve accessibility of the Aggie platform

### ISSUE THREAD

**Issue by karpawich** [TID-Lab/aggie#334]
Since Aggie was written, the software development community has created a number of standards and tools for accessibility. Aggie should be compared against these standards through available tools so that its accessibility can be evaluated and improved.
